app/src/main/res/layout/activity_create_user.xml

```diff
@@ -4,10 +4,10 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".user.CreateUserActivity">
+    tools:context=".CreateUserActivity">

```

Comment:
Some hardcoded strings also in this layout.

We could use textEmail in the inputType when having to write an email address.